package.json

```diff
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@aeternity/aepp-calldata": "^1.5.1",
-    "@aeternity/aepp-sdk": "^13.2.2",
+    "@aeternity/aepp-sdk": "github:aeternity/aepp-sdk-js#feat/jwt-signature",
```

Comment:
We would definitely need a proper release from the sdk, in order to include it in our release.